tests/io/v3/vm/permissions/test_schema.py

```diff
@@ -0,0 +1,12 @@
+'''
+Testing the Permissions Schema
+'''
+from tenable.io.v3.vm.permissions.schema import PermissionSchema
+
+
+def test_schema():
+    schema = PermissionSchema()
+    payload = schema.dump(schema.load(
+        {'acls': [{'type': 'user', 'id': '2236706', 'permissions': 64}]}))
+    assert payload == {
+        'acls': [{'type': 'user', 'id': 2236706, 'permissions': 64}]}
```

Comment:
We can make this object as a constant and avoid repetation

Reply:
Updated